app/views/admin/chapters/index.html.haml

```diff
@@ -0,0 +1,45 @@
+%section#banner
+  .row
+    %h1 Chapters
+    = link_to t('.view_all_chapters'), admin_chapters_path
+    &nbsp;
+    = link_to t('.view_active_chapters'), admin_chapters_path(active: true)
+    &nbsp;
+    = link_to t('.view_inactive_chapters'), admin_chapters_path(active: false)
+    .row
+      %hr
```

Comment:
similar to the comment above, as a row is made up of column elements, `hr` should be within a ` .large-12.columns`